maven-surefire-plugin/src/site/apt/examples/junit-platform.apt.vm

```diff
@@ -526,6 +526,9 @@ else
 
     * To exclude <<<engines>>>, use <<<excludeJUnit5Engines>>>.
 
+   Be aware that this feature reserve system properties <<<includejunit5engines>>> and <<<excludejunit5engines>>>
```

Comment:
`reserve` should become `reserves`, right?

Reply:
Done